.eslintrc.cjs

```diff
@@ -60,8 +60,14 @@ module.exports = {
 		{
 			files: ["**/*.{yml,yaml}"],
 			parser: "yaml-eslint-parser",
-			extends: ["plugin:yml/base", "plugin:yml/prettier"],
+			extends: ["plugin:yml/recommended", "plugin:yml/prettier"],
```

Comment:
Most of the rules enabled below are also enabled in `plugin:yml/standard`. Which I'm only learning about today, reviewing this PR 😅 sorry for suggesting limiting to `recommended` before. Is there any reason not to use `plugin:yml/standard`?

Reply:
It had couple rules that would conflict with prettier, since we're also using prettier preset we're good to go.

https://github.com/ota-meshi/eslint-plugin-yml/blob/master/src/configs/prettier.ts

Reply:
Woop!